src/DynamoCore/Graph/Nodes/NodeModel.cs

```diff
@@ -503,36 +503,81 @@ public string DictionaryLink
+                    finalLink += "Create/";
+                    break;
+                case Configurations.CategoryGroupQuery:
+                    finalLink += ObtainURL(category.Substring(0, i));
+                    finalLink += "Query/";
+                    break;
+                default:
+                    finalLink += ObtainURL(category);
+                    finalLink += "Action/";
+                    break;
+            }
+            finalLink += this.ShortenName;
 
-                int i = category.LastIndexOf(Configurations.CategoryDelimiterString);
-                switch (category.Substring(i + 1))
+            // Check if the method has overloads
+            IEnumerable<FunctionDescriptor> descriptors = libraryServices.GetAllFunctionDescriptors(CreationName.Split('@')[0]);
+            if (descriptors != null && descriptors.Count() > 1)
+            {
+                // If there are overloads
+                string parameters = "(";
+                IEnumerable<Tuple<string, string>> inputParameters = null;
+
+                foreach (FunctionDescriptor fd in descriptors)
                 {
-                    case Configurations.CategoryGroupAction:
-                        finalLink += ObtainURL(category.Substring(0, i));
-                        finalLink += "Action/";
-                        break;
-                    case Configurations.CategoryGroupCreate:
-                        finalLink += ObtainURL(category.Substring(0, i));
-                        finalLink += "Create/";
-                        break;
-                    case Configurations.CategoryGroupQuery:
-                        finalLink += ObtainURL(category.Substring(0, i));
-                        finalLink += "Query/";
-                        break;
-                    default:
-                        finalLink += ObtainURL(category);
-                        finalLink += "Action/";
+                    if (fd.MangledName == CreationName) // Find the function descriptor among the overloads and obtain their parameter names
+                    {
+                        inputParameters = fd.InputParameters;
                         break;
+                    }
+                }
+                // Convert the parameters into a valid Dictionary URL format, e.g. (x_double-y_double-z_double)
+                if (inputParameters != null)
+                {
+                    int parameterCount = inputParameters.Count();
+                    for (int k = 0; k < parameterCount - 1; k++)
+                    {
+                        parameters += inputParameters.ElementAt(k).Item1 + "_" + inputParameters.ElementAt(k).Item2 + "-";
+                    }
+                    // Append the last parameter without the dash and with the close bracket
+                    parameters += inputParameters.ElementAt(parameterCount - 1).Item1 + "_" + inputParameters.ElementAt(parameterCount - 1).Item2 + ")";
```

Comment:
Would it be better if we do not use `foreach` for this loop as stated in your previous comment, and use the `count` to obtain `lastInputParam`?